pages/linux/select.md

```diff
@@ -7,9 +7,13 @@
 
 `select {{word}} in {{apple}} {{orange}} {{pear}} {{banana}}; do echo ${{word}}; done`
 
-- Create a menu for picking a file or folder from the current directory:
+- Create a menu from the output of another program:
 
-`select {{file}} in *; do echo ${{file}}; done`
+`select {{line}} in $({{program_to_run}}); do echo ${{line}}; done`
+
+- Specify the prompt string for select and create a menu for picking a file or folder from the current directory:
```

Comment:
```suggestion
- Specify the prompt string and create a menu for picking a file or folder from the current directory:
```

Reply:
PS3 is a dedicated prompt string and is only used by `select`, which is why I chose to specify it. Can you elaborate why it shouldn't be mentioned?

Reply:
It's a page that documents `select`, so I thought it would be quite obvious that the prompt is also for `select`.

If you feel like it should be highlighted
```suggestion
- Specify the prompt string for `select` and create a menu for picking a file or folder from the current directory:
```

Reply:
Other prompts like PS1 and PS2 have multiple purposes while while PS3 and PS4 are single purpose. It's why I felt that it should be specified. A third opinion would probably be useful here.